package.json

```diff
@@ -5,7 +5,7 @@
     "test": "jest",
     "test:coverage": "jest --coverage",
     "test:acceptance": "testcafe chrome test/acceptance/",
-    "postinstall": "lerna bootstrap --no-ci && lerna run build",
+    "postinstall": "lerna bootstrap --hoist --no-ci && lerna run build",
```

Comment:
This is the main change that make things work.
`lerna bootstrap` behaves differently with this version of Node and without the `--hoist` parameter it doesn't save anything in the root `./node_modules/.bin`, meaning that without this change also the sequent `lerna run build` fails straight away because of "command not found".